website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -167,6 +168,35 @@ Avoid using different density settings for parent and child rows.
 
 !!!
 
+### Expand/Collapse All Button
+The Expand/Collapse All button allows users to expand or collapse all rows, including nested rows. It provides quick access to more content but may impact readability when content is long or detailed.
+
+### Interactions
+
+#### Default state
+
+The Advanced Table supports any combination of expanded or collapsed rows on load. The button reflects the initial state.
+
+![Expand all button with an expand icon.](/assets/components/table/advanced-table/expandable-rows-expand-all.png)
+<Doc::ImageCaption @text="Expand All if any rows are collapsed."/>
```

Comment:
```suggestion
<Doc::ImageCaption @text='“Expand All” button displays if any rows are currently collapsed'/>
```

Reply:
If desired, add "The" to the beginning of the above phrase to make a complete sentence and then add back a period at the end.

Reply:
You could also remove the quotes around the button names, if you wish, but I thought they helped provide clarity.